code/modules/mob/mob_defines.dm

```diff
@@ -200,3 +200,6 @@
 
 	/// The location our runechat message should appear. Should be src by default.
 	var/atom/runechat_msg_location
+
+	/// The datum receiving keyboard input. src by default.
+	var/datum/input_focus = null
```

Comment:
Does this mean you could have a mob remote controlling another mob without actually having to inhabit that mob? :eyes: Something like a borer feeding their inputs to their host.

Reply:
It would be possible with some work because the movement loop calls the client's `Move` proc, which rely on its current `mob`, not one that is explicitly passed